src/components/Nav/Nav.Item.jsx

```diff
@@ -35,6 +35,7 @@ export class NavItem extends React.Component {
       location,
       strict,
       to,
+      'data-bypass': this.props['data-bypass'],
```

Comment:
If you want to follow the destructuring above, you can rename it in place:

```
const { disabled, exact, isActive, location, strict, to, 'data-bypass': dataBypass } = this.props


//...
to,
'data-bypass': dataBypass,
}
```

This is how we do it with `data-cy`